Codeinwp/templates-patterns-collection
# Fix/issue 109

### Summary
Solves the refresh JS error.
This was introduced as part of the changes from here: https://github.com/Codeinwp/templates-patterns-collection/pull/79

### Test instructions
1. Create a clean instance with Neve and TPC
2. Go to Page Templates tab (under Appearance -> Neve Options > Starter Sites)
3. Refresh the page (or just go to another page, and Go Back to Page Templates tab)
4. The page should load properly, previously it was not and was triggering an error.



Closes Codeinwp/templates-patterns-collection#109.
